tsa-core/src/main/kotlin/org/ton/bytecode/TvmArtificialInstructions.kt

```diff
@@ -45,6 +46,18 @@ data class TsaArtificialActionPhaseInst(
     }
 }
 
+data class TsaArtificialOnOutMessageHackInst(
```

Comment:
Please give this a more understandable name. For example, `TsaArtificialOnOutMessageEntry` (or exit, depending on the logic)